sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -65,1050 +45,387 @@ def __init__(
+         a ReverseGeocodingBatchRequestBody type or a IO[bytes] type. Required.
+        :type reverse_geocoding_batch_request_body:
+         ~azure.maps.search.models.ReverseGeocodingBatchRequestBody or IO[bytes]
+        :return: GeocodingBatchResponse
+        :rtype: ~azure.maps.search.models.GeocodingBatchResponse
+        :raises ~azure.core.exceptions.HttpResponseError:
         """
-        batch_items = [{"query": f"?query={query}"} for query in search_queries] if search_queries else []
-
-        batch_result = self._search_client.reverse_search_address_batch_sync(
-            batch_request={"batch_items": batch_items},
+        result = self._search_client.get_reverse_geocoding_batch(
+            reverse_geocoding_batch_request_body,
             **kwargs
         )
-
-        return batch_result
+        return result
 
     @distributed_trace
-    def begin_reverse_search_address_batch(
-        self,
-        search_queries: List[str],
-        **kwargs: Any
-    ) -> LROPoller[ReverseSearchAddressBatchProcessResult]:
-        """*Begin Search Address Reverse Batch API Request**
-
-        Sends batches of reverse geocoding requests.
-        The method returns a poller for retrieving the result later.
-
-        The Search Address Batch API sends batches of queries to `Search Address Reverse API`.
-        You can call Search Address Reverse Batch API to run either asynchronously (async) or
-        synchronously (sync). The async API allows caller to batch up to **10,000** queries and sync
-        API up to **100** queries.
-
-        :param search_queries:
-            The list of search fuzzy queries/requests to process.
-            The list can contain a max of 10,000 queries and must contain at least 1 query.
-        :type search_queries:
-            List[str]
-        :return:
-            The results of reverse search batch request.
-        :paramtype:
-            ~azure.core.polling.LROPoller[~azure.maps.search.models.ReverseSearchAddressBatchProcessResult]
-        :raises:
-            ~azure.core.exceptions.HttpResponseError:
+    def get_polygon(
+            self,
+            *,
+            coordinates: List[float],
+            view: Optional[str] = None,
+            result_type: Union[str, _models.BoundaryResultTypeEnum] = "countryRegion",
+            resolution: Union[str, _models.ResolutionEnum] = "medium",
+            **kwargs: Any
+    ) -> _models.Boundary:
+        """Use to get polygon data of a geographical area shape such as a city or a country region.
+
+        The ``Get Polygon`` API is an HTTP ``GET`` request that supplies polygon data of a geographical
+        area outline such as a city or a country region.
+
+        :keyword coordinates: A point on the earth specified as a longitude and latitude. Example:
+         &coordinates=lon,lat. Required.
+        :paramtype coordinates: list[float]
+        :keyword view: A string that represents an `ISO 3166-1 Alpha-2 region/country code
```

Comment:
rename all the arguments to `localized_map_view`. Please refer to the original code.

Reply:
And we should also define `LocalizedMapView` type. Please refer to the original code as well.